doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -714,7 +714,12 @@ nRF RPC libraries
 Other libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`lib_date_time` library:
```

Comment:
Not related to this file but new retry logic should be mentioned in few places of the library documentation.